.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+---
+
+<!--
+Thanks for filing a 🙋 feature request 😄!
+-->
+
+<!-- Provide a general summary of the issue in the title above -->
+
+**Describe the 🙋 feature request**
```

Comment:
These two sections sound quite similar to me. Perhaps this section could be titled "Describe the problem you are trying to solve".  Maybe "problem" isn't the right word, but generally steer the reporter to explaining **why** they want the `--do-awesome-things` flag.